openrtb_ext/request_wrapper.go

```diff
@@ -95,6 +95,12 @@ func (rw *RequestWrapper) GetImp() []*ImpWrapper {
 
 func (rw *RequestWrapper) SetImp(imps []*ImpWrapper) {
 	rw.impWrappers = imps
+	imparr := make([]openrtb2.Imp, len(imps), len(imps))
+	for i, imp := range imps {
```

Comment:
Nitpick: `imp` is actually an `impWrapper`.

Reply:
Thanks